Decisions.TruBot/Data/TruBotProcess.cs

```diff
@@ -73,7 +70,7 @@ internal static TruBotProcess GetTruBotProcessByBotId(string truBotId)
             return orm.Fetch(new WhereCondition[]
             {
                 new FieldWhereCondition("BotId", QueryMatchType.Equals, truBotId)
-            }).FirstOrDefault();
+            }).FirstOrDefault()!;
```

Comment:
I don't think I've seen this syntax...what is it?

Reply:
I believe it suppresses/forgives a nullable warning